modules/slack/resources/schema.yaml

```diff
@@ -0,0 +1,47 @@
+title: Schema for Jenkins Slack Notification plugin configuration
+type: object
+additionalProperties: false
+
+properties:
+  team_subdomain:
```

Comment:
> There could be on-premise installations in future.

Premature optimisation fan?)

Feel free to fix this upstream https://github.com/jenkinsci/slack-plugin, I'm strongly opposed to adding some parsing logic here for this.